src/components/form/form.types.ts

```diff
@@ -154,21 +154,54 @@ export interface FormComponentOptions {
     props?: Record<string, any>;
 }
 
-export interface FormLayoutOptions {
+export interface FormLayoutOptions<T = FormLayoutType.Default> {
```

Comment:
Considering the change on line 138 (see below), I was a little worried this might be a breaking change, but I tried removing the `<any>` and the prod-build was still successful, so it seems to me this change is non-breaking.

```diff
-    layout?: FormLayoutOptions;
+    layout?: FormLayoutOptions<any>;
```

(Just adding this as a bit of a record for posterity, in case it turns out this actually does break stuff 😄)